lib/src/metta/runner/stdlib.rs

```diff
@@ -707,6 +707,33 @@ impl Grounded for PrintlnOp {
     }
 }
 
+#[derive(Clone, PartialEq, Debug)]
+pub struct LogOp {}
+
+impl Display for LogOp {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "log!")
+    }
+}
+
+impl Grounded for LogOp {
+    fn type_(&self) -> Atom {
+        Atom::expr([ARROW_SYMBOL, ATOM_TYPE_UNDEFINED, ATOM_TYPE_UNDEFINED, ATOM_TYPE_UNDEFINED])
```

Comment:
@vsbogd, is there a type for String?  I.e. could I write something like
```rust
Atom::expr([ARROW_SYMBOL, STRING_TYPE, Atom::var("a"), Atom::var("a")])
```
?

Reply:
Never mind, I'm almost sure the answer is no because `String` is defined in https://github.com/trueagi-io/hyperon-experimental/blob/09dbcb688c33299da334c9660b977740cd5ac572/python/hyperon/stdlib.py#L42

Reply:
Oh, could it be
```rust
Atom::expr([ARROW_SYMBOL, Atom::sym("String"), Atom::var("a"), Atom::var("a")])
```
?

Reply:
Yes, for the strings which are recognized by a Python tokenizer it is `Atom::sym("String")`.

Rust `stdlib.rs` doesn't provides a string token yet. The reason it is not yet implemented is we were thinking on a way to uniformly support basic types (like `Int`, `Float` and `String`) across Python and Rust libraries. But we stopped on a state when Python and Rust has separate and incompatible atoms for such basic types.

It is implemented such as Python tokenizer can redefine some tokens from root Rust tokenizer. You can see that integer token is defined in both `stdlib.rs` and `stdlib.py`. Basically when MeTTa code interpreted from Python the `(42)` atom contains a reference to the Python object `int(42)`. And if you run the same program from Rust interpreter the same atom `(42)` contains a Rust integer.